development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
+[age_groups]
+
+[[under_7]]
+desc            = "Under 7"
+min_age         = 0
+max_age         = 7
+discount        = 999
```

Comment:
This is a little kludgy; since badge costs cannot ge below 0, we're just using $999 discount to mean "free attendance".  This would probably be clearer if we made it a constant or something.  ConfigObj natively lets us say ```discount = %(free)s" and other things along those lines.